test/test_ops.py

```diff
@@ -454,7 +454,7 @@ def expected_fn(self, x, weight, offset, bias, stride=1, padding=0, dilation=1):
         return out
 
     def get_fn_args(self, device, contiguous):
-        batch_sz = 1
+        batch_sz = 33
```

Comment:
Why not test both batch_sz 1, 33 and some larger value (in case that magic number changes)?

Reply:
I would like to change this test to be faster actually -- in it's current state it is the slowest test in torchvision (takes 30s to run). Ideally, I would like to only do gradcheck on a smaller tensor, but still check for correctness in the large-tensor case. My plan was to open an issue to improve this in the future